sdk/eventhub/azure-eventhub/azure/eventhub/_pyamqp/aio/_transport_async.py

```diff
@@ -558,7 +557,7 @@ async def _read(self, n, buffer=None, **kwargs):  # pylint: disable=unused-argum
             return view
         except asyncio.TimeoutError as te:
             raise ConnectionError('Receive timed out (%s)' % te)
-        except OSError as e:
```

Comment:
So we are happy for OSErrors to be propagated from here rather than caught? I'm guessing this will be caught higher up in the `read()` or `receive_frame()` methods?